test/hello_world.test.js

```diff
@@ -1,9 +1,35 @@
 var test = require('tape');
 var HelloWorld = require('../lib/index.js');
+var HW = new HelloWorld();
 
-test('wave hi', function(t) {
-  var HW = new HelloWorld();
+test('wave', function(t) {
   var hello = HW.wave();
-  t.equal(hello, 'howdy world!', 'output of HelloWorld.wave');
+  t.equal(hello, 'howdy world', 'output of HelloWorld.wave');
   t.end();
-})
\ No newline at end of file
+});
+
+test('shout', function(t) {
+  HW.shout('rawr', {}, function(err, shout) {
+    if (err) throw err;
+    console.log(shout);
+    t.end();
+  });
+});
+
+test('shout error - non string phrase', function(t) {
+  HW.shout(4, {}, function(err, shout) {
+    t.ok(err);
```

Comment:
I would expect `err` to be here, but we are throwing within `hello_world.cpp` instead of returning an error message. Not sure how to do this. @springmeyer